src/amo/sagas/reviews.js

```diff
@@ -50,6 +56,9 @@ import type {
   UpdateAddonReviewAction,
 } from 'amo/actions/reviews';
 
+// Number of millesconds that a message should be flashed on screen.
+export const FLASH_MESSAGE_DURATION = 2000;
```

Comment:
It seems that this only applies to the SAVED messages duration.?

Reply:
Correct. I can rename it.